src/fragment-generation-utils.js

```diff
@@ -1033,12 +1119,12 @@ const containsBlockBoundary = (range) => {
   if (!walker) {
     return false;
   }
-  const map = createForwardOverrideMap(walker);
+  const isFinished = new Set();
```

Comment:
By convention, I usually expect `isFoo` to be (or return) a boolean. Maybe `visitedNodes`?